tests/modules/nf-core/sourmash/sketch/test.yml

```diff
@@ -5,6 +5,6 @@
     - sourmash/sketch
   files:
     - path: output/sourmash/test.sig
-      md5sum: 4b0dee307e35fc670cd0d416321e4961
+      md5sum: c23a5a251d2b6babba8ae0e3905e8d04
     - path: output/sourmash/versions.yml
```

Comment:
```suggestion
```

Checks for versions.yml now happen automatically

Reply:
Could you also remove the `md5sum`? Github doesn't allow me to suggest a change there